### DISCUSSION TITLE
Getting this error : Failed to start gauge API: Error occurred while waiting for runner process to finish.

### DISCUSSION THREAD

**Discussion by tuna81** [getgauge/gauge#2292] [General]
i am getting this 
Failed to start gauge API: Error occurred while waiting for runner process to finish.
"error: exit satus 1"

what should i do? does anyone know this

<img width="788" alt="Ekran Resmi 2022-10-05 21 02 13" src="https://user-images.githubusercontent.com/108681919/194129982-6de9efb7-d7a9-44a7-a16a-21849504bc8f.png">

**Comment by zabil** [comment]
The screenshot you posted shows compilation java code compilation errors. Gauge will run once you fix those.

**Reply by tuna81** [reply]
I couldn't find the compilation errors you mentioned. project working from ide, there is not any problem. but i try "gauge run specs" on terminal then i am getting this error.

And 
I get the same result on terminal when I try with this code I got from ide


<img width="1431" alt="Ekran Resmi 2022-10-10 14 40 13" src="https://user-images.githubusercontent.com/108681919/194858586-3bd4508b-c61f-4620-a4c3-752404054ec2.png">

**Comment by zabil** [comment]
@tuna81 The first few lines of the first screenshot you posted show's the compilation errors. It looks like you are running `gauge run specs` on the command line on a java project after adding selenium libraries in the IDE. This is not supported. To run gauge tests from the command line, please use a build tool like maven or gradle. You can create a sample gauge [maven](https://maven.apache.org) project with the following command 

```
gauge init java_maven
```
